src/init.jl

```diff
@@ -1,10 +1,10 @@
-using Scratch
 using REPL
+import PlotlyJS
```

Comment:
@isentropic It looks like this caused a test failure because PlotlyJS is not a dependency. How does that work? I thought PlotlyJS is always loaded by default?

Reply:
> When I run `import Plots; plotly(); plot(1:10)` in Pluto, does that load the PlotlyJS package?

No, only `import Plots; plotlyjs(); plot(1:10)` will load PlotlyJS, `plotly()` is special in that it is the only backend that is usable without any 3rd party (julia) package, but it can only display to a web browser, while `plotlyjs` has a standalone display via Blink.jl